### PR TITLE
Prepare 0.17.0 release

### LINKED ISSUE #387: Release a version with min/max on histogram
Min/max was added in https://github.com/open-telemetry/opentelemetry-proto/pull/279 after the 0.16.0 release, which is, at the time of opening this issue, the latest release. Could a new release be made so that consumers of this project can use the min/max that was added?

### PR DESCRIPTION
**Pull request by arminru**
Resolves #387.

### CODE REVIEW

**Review by tigrannajaryan — APPROVED** [review]

**Review by srikanthccv — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
If we are ready for 0.17.0 release then it does not need to wait for https://github.com/open-telemetry/opentelemetry-proto/pull/376 approval. We can make another release after that just to declare Logs stable (there is no proto changes anyway).

**Review by bogdandrutu — APPROVED** [review]

**Comment by arminru** [comment]
@tigrannajaryan Looks like #376 would merge today or tomorrow so we can do it at once.

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +20,15 @@ Full list of differences found in [this compare](https://github.com/open-telemet
 
 * Remove if no changes for this section before release.
 
+## 0.17.0 - 2022-05-04
```
Comment (arminru) [inline CHANGELOG.md:23]:
```suggestion
## 0.17.0 - 2022-05-06
```

**Comment by tigrannajaryan** [comment]
@arminru let's release without merging https://github.com/open-telemetry/opentelemetry-proto/pull/376
I will do anothe release next weke,

**Pull request merged**